cmd/recv.go

```diff
@@ -199,6 +199,21 @@ func recvAction(cmd *cobra.Command, args []string) {
 					bail("Read zip error: %s", err)
 				}
 
+				// calculate the uncompressed size of the contents of the zip
+				var actualUncompressedSize uint64
+				var fileCount int
+				for _, f := range zr.File {
+					actualUncompressedSize += f.FileHeader.UncompressedSize64
```

Comment:
For a malicious zipfile I don't think we can trust the value of `UncompressedSize64`